test/system/field_slips_test.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require "application_system_test_case"
+
+class FieldSlipsTest < ApplicationSystemTestCase
+  setup do
+    @field_slip = field_slips(:one)
```

Comment:
```suggestion
    @field_slip = field_slips(:field_slip_one)
```
Fixes 
```ruby
ERROR FieldSlipsTest#test_visiting_the_index (0.75s)
Minitest::UnexpectedError:         StandardError: No fixture named 'one' found for fixture set 'field_slips'
            test/system/field_slips_test.rb:7:in `block in <class:FieldSlipsTest>'
```

Reply:
Weird.  The tests pass fine for me locally without this change which I totally don't understand.  Thanks for fixing it.

Reply:
Oh, I thought you pushed it already.  I just did the same change locally and have pushed it.